### DISCUSSION TITLE
Seek Code

### DISCUSSION THREAD

**Discussion by TheNumber122** [eritislami/evobot#781] [Q&A]
I need Command to seek Music like Rythm or anything like else :L Give me command for
seek.js

**Comment by GasparMPereira** [comment]
the rythm bot dont use the same framework

[a comment by an automated account is not shown]

**Comment by wolf-yuan-6115** [comment]
> I need Command to seek Music like Rythm or anything like else :L Give me command for
> seek.js

Rythm is not using Node js

**Comment by GasparMPereira** [comment]
I think rythm use https://www.npmjs.com/package/@2pg/music
Or 
https://preview.npmjs.com/package/distube
i only know ytdl dont have seek code

**Comment by wolf-yuan-6115** [comment]
> I think rythm use https://www.npmjs.com/package/@2pg/music
> Or 
> https://preview.npmjs.com/package/distube
> i only know ytdl dont have seek code

Rythm is not using Node js, I said that... Rythm is written in **Java** , not Java**Script**

[a comment by an automated account is not shown]
